src/components/Editable.tsx

```diff
@@ -346,6 +346,20 @@ const Editable = ({
         ),
       )
 
+      if (newValue.toLowerCase().indexOf(' new thought') > 0) {
+        const [prev, next] = newValue.split('new thought')
+        newValue = prev.trim()
+
+        setTimeout(() => {
+          dispatch(
+            newThought({
+              at: rootedParentOf(state, path),
```

Comment:
You should omit `at`. It will create the new thought after the cursor thought by default. Creating the new thought at the end of the parent doesn't make sense.

Reply:
got it